Mathlib/AlgebraicTopology/SimplexCategory.lean

```diff
@@ -352,13 +352,13 @@ section Skeleton
 /-- The functor that exhibits `SimplexCategory` as skeleton
 of `NonemptyFinLinOrd` -/
 @[simps obj map]
-def skeletalFunctor : SimplexCategory ⥤ NonemptyFinLinOrd.{v} where
-  obj a := NonemptyFinLinOrd.of <| ULift (Fin (a.len + 1))
-  map f := ⟨fun i => ULift.up (f.toOrderHom i.down), fun i j h => f.toOrderHom.monotone h⟩
+def skeletalFunctor : SimplexCategory ⥤ NonemptyFinLinOrd where
+  obj a := NonemptyFinLinOrd.of (Fin (a.len + 1))
```

Comment:
Is the argument here that if you want `NonemptyFinLinOrd.{v}`, you cam compose with something like `NonemptyFinLinOrd.ulift.{v} : NonemptyFinLinOrd.{u} ⥤ .NonemptyFinLinOrd {max u v}`?

Reply:
Yes.